### PR TITLE
Increase <~ operator precedence

### PR DESCRIPTION
**Pull request by neilpa**
Enables assignment of the returned disposable to an optional var without parenthesizing the entire binding expression.

### CODE REVIEW

File: ReactiveCocoa/Swift/Property.swift [hunk 1]
```diff
@@ -161,7 +161,9 @@ extension MutableProperty: SinkType {
 
 infix operator <~ {
 	associativity right
-	precedence 90
+
+	// Binds tighter than assignment but looser than everything else.
+	precedence 91
```
Comment (neilpa) [inline ReactiveCocoa/Swift/Property.swift]: I could bump this to 95 to leave some space between  but wasn't sure.

Comment (jspahrsummers) [inline ReactiveCocoa/Swift/Property.swift]: Maybe 92 or 93? Changing precedence later is technically a breaking change.

Also, can you document that this is (intentionally) less than `|>`?

**Comment by jspahrsummers** [comment]
:+1:

**Comment by neilpa** [comment]
:boom:

**Pull request merged**